.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+name: Deploy to Pages
+
+on:
+  push:
+    branches: ["master"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout files
+        uses: actions/checkout@v4
+      - name: Build static assets
+        shell: bash
+        run: make
+      - name: Copy static assets to "static" folder
+        shell: bash
+        run: |
+          mkdir static
+          find . "(" -name "*.html" -o -name "*.css" -o -name "*.js" ")" -not -path "./static/*" -not -path "./.github/*" -exec cp {} static \;
```

Comment:
That's a pretty long command. Can we simplify somehow?

Reply:
Maybe we can put this logic into make too, `make deploy-static`. I really don't like putting too much complexity on CI/CD files.

Another reason: if we want to serve the site locally, we will need to do this, right?

Since we renamed the paths from long.setup and short.setup.

Reply:
I actually didn't think of and didn't test serving it locally.

Good catch!

Reply:
Resolvido